packages/graphback-core/tests/createInMemoryFilterPredicateTest.ts

```diff
@@ -871,14 +871,14 @@ describe('createInMemoryFilterPredicate', () => {
       name: {
         endsWith: 'Simpson'
       },
-      and: {
+      and: [{
```

Comment:
General comment about in memory filter: 

Good to see this file modified, should we also update this condition:  

https://github.com/aerogear/graphback/blob/c0d25d556583a3df7fd85e1fbaf0a43a245b2586/packages/graphback-core/src/runtime/createInMemoryFilterPredicate.ts#L146
?

Reply:
Yes, I'll check that out.